### PR TITLE
Add clipboard focus listener

### PR DESCRIPTION
**Pull request by dorroddorrod**
Add an event listener for "focus" action to sync the local clipboard to the remote clipboard without clicking the sync button

### CODE REVIEW

**Comment by tinyzimmer** [comment]
Just want to say thank you **so** much for the PR. I've been desperate to get more hands on this code base :smile:.

I'm a bit busy today and the following days, but I should have time to look this weekend.

File: ui/app/src/layouts/MainLayout.vue [hunk 1]
```diff
@@ -314,6 +314,7 @@ export default {
       this.onClickLogin()
       this.pushIfNotCurrent('login')
     }
+    await this.onFocusSyncRemoteClipboardListener()
```
Comment (tinyzimmer) [inline ui/app/src/layouts/MainLayout.vue:317]: Does this to be await?

**Comment by tinyzimmer** [comment]
After giving it a quick look, I guess my main question is if the event listener needs to be removed at any point to keep it from being registered multiple times? The effects of that seem benign, but could cause problems if it's an issue. Did you test it out at all on your own by chance?

File: ui/app/src/layouts/MainLayout.vue (hunk 1 above)
Comment (dorroddorrod) [reply ui/app/src/layouts/MainLayout.vue:317]: Not really.. i'll remove it

**Comment by dorroddorrod** [comment]
I did test it on my local machine and it works. not sure if this listener can registered multiple times. would you mind to give it a test ?

**Comment by tinyzimmer** [comment]
Will play with it a bit later

**Comment by dorroddorrod** [comment]
Any updates ?

**Comment by tinyzimmer** [comment]
Sorry for the delay - I still think there might be a better way to do this, but it does seem to work fine and not cause too many issues. I think whenever I get around to rewriting the front-end this will become some sort of top-level run-once sorta thing.

**Pull request merged**